src/components/HomePage/MainView.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import logo from './nijobs.png';
+
+import SearchArea from "./SearchArea/SearchArea";
+import ShowMoreButton from "./ShowMoreButton";
+import InfoBox from "./QuickInfoArea/InfoBox";
+
+import homePageStyles from './HomePage.module.css';
```

Comment:
Debate: Using CSS Modules vs Using the new MUI styles hooks, which would be best?

Reply:
I think both have their place. Currently I am using the MUI hooks for overrides in the MUI components themselves, while the CSS Modules serve as general CSS rules for the application (div placement, grids, etc)

Reply:
Makes sense, this way the CSS Modules maintain their broader scope and our CSS does not end up like 🍝, while taking advantage of MUI overrides 😄 

Still, might be better to document these decisions somewhere, I'm thinking throwing something on the README for now would be good - what do you think? We should track these as issues or somewhere else too, if we don't do them straight away (don't have a Jira, PivotalTracker or simillar for now so GitHub Issues will do).

Should also look into starting some less specific CSS Modules - I'm certain some placement, sizing, or coloring ones will eventually be useful, and this would improve modularity - a concern to keep in mind for the future.

Reply:
I think we should stick to the sizing/coloring conventions from MUI Theme directly, and in the CSS Modules only change higher-level visuals (such as alignment, for example)

Reply:
Fair point, was just listing some examples off the top of my head, but agreed